joomla/joomla-cms
# [4.3] [Guided Tours] Fancy select fields are broken and not accessible in tours

Pull Request for Issue #40211 .

### Summary of Changes

Categories and tags are fancy select fields. Although tags can be selected in tours, categories can not.
This PR fixes categories and improves the tags selection fields.
Fields can be tabbed through, categories and tags selected (to select a category or tab, use the arrow keys) or added, as intended.

### Testing Instructions

Take the article tour up to the category field.
Tab through to see that the popup and the field keep focus and you can go from one to the other.
Once it has focus, the field's dropdown should open (if you tab through or click on the field). 
A category can be entered. To access already existing categories, use the arrow keys. 
Press enter on the category and the article will change to the new category.
You will loose the 'back' button in the popup, since a refresh has been made.

Go down and test the tags the same way. This time there is no refresh when selecting tags.

Take the tour for guided tours. Check the component parameter as well.

### Actual result BEFORE applying this Pull Request

You cannot select or add a category.

### Expected result AFTER applying this Pull Request

You can select or add a category.

### Link to documentations
Please select:
